tests/cases/helpers/case_helpers_care_platform.yaml

```diff
@@ -1714,4 +1714,26 @@ CarePartnerEditParticipantNavigation:
       Strategy: xpath
       Id: $PAGE.care_platform_participant_records.edit_info_button$
     - Type: sleep
-      Time: 1
\ No newline at end of file
+      Time: 1
+
+CarePartnerToastifyValidation:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+    - Type: get_text
```

Comment:
could you add a wait_for before the get_text action? the test could fail easily if the toast doesn't appear right away.